apps/api/src/routes/serviceRoute.ts

```diff
@@ -0,0 +1,77 @@
+import { Hono } from "hono"
+import { PublicContextVariables } from "@alittlebyte/api/utils/types"
+import { zValidator } from "@hono/zod-validator"
+import { idValidator, serviceValidator } from "@alittlebyte/common/validators"
+import { z } from "zod"
+
+export const servicesRouter = () =>
+	new Hono<{ Variables: PublicContextVariables }>()
+		.get("/", async ({ json, var: { prisma } }) => {
+			const services = await prisma.service.findMany()
+			return json({ message: services }, 200)
```

Comment:
The status code by default is 200
and the list of services isn't a message but a data
```suggestion
			return json({ data: services })
```